protocol/app/upgrades/v0.2.2/upgrades.go

```diff
@@ -0,0 +1,160 @@
+package v0_2_2
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
+	clobmodulekeeper "github.com/dydxprotocol/v4-chain/protocol/x/clob/keeper"
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+func CreateUpgradeHandler(
```

Comment:
For my own understanding, do we plan on deleting any of the migration/upgrade logic in the future? Logic including this handler and other dependent methods like `UnsafeSetClobPair`